src/app/app-routing.module.ts

```diff
@@ -60,30 +66,38 @@ export const componentRoutes = [
   { path: 'list', component: ListPage },
   { path: 'modal', component: ModalPage },
   { path: 'password', component: PasswordPage },
-  { path: 'progressbar', component: ProgressbarPage },
+  { path: 'progress-bar', component: ProgressbarPage },
```

Comment:
I think that @bbogdanov had some changes related to the change of paths lately?

Reply:
@bdryanovski 

At first I thought that as well but I remember the change actually was inside the `not-ready` component. Nothing to worry about here.